### PR TITLE
Removes references from absinthe

### PR DESCRIPTION
**Pull request by Niknakflak**
#16147 Snuck in some references to shitty players last month.  This is not apart of the grandfather clause and references to shitty players is a meme to move away from.

### CODE REVIEW

**Comment by KazeEspada** [comment]
Alchohol always references griefers

**Comment by Niknakflak** [comment]
All the references were grandfathered in, these are new

**Comment by optimumtact** [comment]
You should replace the names with something else like Taylor and Tailor or something equally boring

**Comment by TechnoAlchemisto** [comment]
not a player reference, stop grudging you absolute baby

**Comment by optimumtact** [comment]
No see, you've made it even more boring now

**Comment by Niknakflak** [comment]
Boring is what I do

**Comment by xxalpha** [comment]
Don't make it a generic brand.

**Comment by Niknakflak** [comment]
Namebrands will take over the galaxy, I'm open to non-reference suggestions

**Comment by xxalpha** [comment]
I suggest you close this PR

**Comment by TechnoAlchemisto** [comment]
This is a boring af pr although it is kind of weird that we have a no player reference policy and this got through tbh

**Comment by Niknakflak** [comment]
This rectifies broken policy

**Comment by TechnoAlchemisto** [comment]
if this somehow gets merged ill give it a nice spicy name

**Comment by coiax** [comment]
It's a reference? Streisand effect, top kek.

File: code/modules/food&drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -236,8 +236,8 @@
 	list_reagents = list("wine" = 100)
 
 /obj/item/weapon/reagent_containers/food/drinks/bottle/absinthe
-	name = "T&T Extra-Strong Absinthe"
-	desc = "An strong alcoholic drink brewed and distributed by Terb & Tonte Inc."
+	name = "Extra-Strong Absinthe"
+	desc = "An strong alcoholic drink."
```
Comment (KorPhaeron) [inline code/modules/food&drinks/drinks/drinks/bottle.dm]: >An strong

Bad PR

Comment (PKPenguin321) [inline code/modules/food&drinks/drinks/drinks/bottle.dm]: Close it

**Comment by Cheridan** [comment]
grandfather clause/? the only grandfather here is you @Niknakflak you old man

**Comment by Bawhoppen** [comment]
look at that thumbs downs

**Comment by coiax** [comment]
@Dorsisdwarf this isn't Controversial, it's universally unpopular.

**Comment by Dorsisdwarf** [comment]
We only have one tag for "opinions are not positive"

**Comment by duncathan** [comment]
I'm not aware of any players named "Terb" or "Tonte".

**Comment by phil235** [comment]
I'm with NikNak on this one, references can be okay, but using shitty players names is just bad.

**Comment by coiax** [comment]
I just downvoted because I thought it was some minor anti-fun thing. But apparently the players were actually shit? Also, this is NikNak's beginnings of contributing to the codebase, so I'm not stamping down

**Comment by xxalpha** [comment]
They are not players, you people are crazy.

**Comment by Niknakflak** [comment]
Please stop filling this PR with your shitty memes

**Comment by KazeEspada** [comment]
ALL BASE ALCHOHOLIC BEVERAGES REFERENCE SHITTERS.
Pete's CUBAN SPICED RUM
DOUBLEBEARDS BEARDED WINE

**Comment by Niknakflak** [comment]
I already responded to that statement

**Comment by Razharas** [comment]
Ok niknakflak points out that indeed theres inconsistency of our policy right here so i reopen it and let cheri as the headcoder clarify on what is to enforce and what not

**Comment by Razharas** [comment]
@Cheridan

**Comment by optimumtact** [comment]
He's not wrong, but now the name is boring and should be something else like Turtleneck & Thompson

**Comment by Iamgoofball** [comment]
Who are Terb and Tonte? They've never played here.

**Comment by Razharas** [comment]
You can relax dudes i dont care one bit i just want to clarify the official stance on one of the policies

**Comment by Niknakflak** [comment]
>  Cheridan: someone make a pr that'll conflict with NikNak's so i can continue ignoring it. i'll give you 20 cheribucks.<Cheridan> someone make a pr that'll conflict with NikNak's so i can continue ignoring it. i'll give you 20 cheribucks.

>TechnoAlchemist: okay

#17226
I am being conspirasized against.

**Pull request closed without merging**